src/reynir_correct/pattern.py

```diff
@@ -1681,16 +1672,16 @@ def create_patterns(cls) -> None:
                     None,
                 )
             )
-
-            # Catch "Hún á (ekki) heiðurinn að þessu.", "Hún hafði (ekki) átt heiðurinn að þessu."
+            # Catch "Ég lét (ekki) gott að mér leiða."
             p.append(
                 (
-                    "heiður",  # Trigger lemma for this pattern
-                    "VP > { VP >> { VP > { 'eiga' } NP > { 'heiður' } } PP > { 'að' } }",
-                    cls.wrong_preposition_heiður_að,
+                    "leiður",  # Trigger lemma for this pattern
+                    "VP > { VP > { 'láta' } PP > { P > { 'að' } 'leiður' } }",
+                    cls.wrong_preposition_að_leiða,
```

Comment:
Suggest ('leiða' | 'leiður')

Reply:
I fixed the whole phrase structure to adapt it to 'leiða' as a verb.